### PR TITLE
A solution for issue #8

### PR DESCRIPTION
**Pull request by gustavgahm**
This pull request contains the option to have translations sorted in the output file.

### CODE REVIEW

File: src/cli/extract.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ const options = cli.parse({
 	output: ['o', 'Path you would like to save extracted strings to', 'dir', process.env.PWD],
 	format: ['f', 'Output format', ['json', 'pot'], 'json'],
 	replace: ['r', 'Replace the contents of output file if it exists (Merges by default)', 'boolean', false],
+	sort: ['s', 'Sort translations in the output file in alphabetical order in', 'boolean', false],
```
Comment (biesbjerg) [inline src/cli/extract.ts]: Extra 'in' in the end

File: src/utils/translation.collection.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export class TranslationCollection {
 		const values = keys.reduce((results, key) => {
 			results[key] = '';
 			return results;
-		}, <TranslationType> {});
+		}, <TranslationType>{});
```
Comment (biesbjerg) [inline src/utils/translation.collection.ts]: Does this not make the lint fail? (npm run lint)

File: src/utils/translation.collection.ts [hunk 3]
```diff
@@ -75,4 +75,18 @@ export class TranslationCollection {
 		return Object.keys(this.values).length === 0;
 	}
 
+	public sort(compareFn?: (a: string, b: string) => number): TranslationCollection {
+
```
Comment (biesbjerg) [inline src/utils/translation.collection.ts]: Remove linebreak

File: src/cli/extract.ts [hunk 4]
```diff
@@ -77,6 +78,10 @@ try {
 		}
 	}
 
+	if(options.sort) {
```
Comment (biesbjerg) [inline src/cli/extract.ts]: if ( ... )

**Comment by biesbjerg** [comment]
Please run `npm run lint` and fix any issues.

**Comment by gustavgahm** [comment]
Sure, I will fix all things above.

**Comment by biesbjerg** [comment]
Thank you for the PR! 

Merged and new package published to NPM!

**Pull request merged**